.codeclimate.yml

```diff
@@ -0,0 +1,19 @@
+---
+engines:
+  checkstyle:
+    enabled: true
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - php
```

Comment:
Can you please modify this for a java code base? :) 

Note: we added the same issue on our 3 PHP repos as well (I am not sure as of right now if those issues are still available)